src/asm_syntax.hpp

```diff
@@ -133,27 +140,32 @@ struct Deref {
     int offset{};
 };
 
+/// Load/store instruction.
 struct Mem {
     Deref access;
     Value value;
     bool is_load{};
 };
 
+/// Legacy, no longer used.
```

Comment:
It does exist in older eBPF programs, and prehaps even in newer programs under certain conditions. It is supported by the Linux runtime and is analyzed correctly (as a function call) in PREVAIL.

I'd say something like: A special instruction for checked access to packets; it is actually a function call, and analyzed as one, e.g. by scratching caller-saved registers after it is performed.

The relevant explanation in the linux verifier is [here](https://elixir.bootlin.com/linux/v5.9.8/source/kernel/bpf/verifier.c#L7315).

Reply:
Updated